charts/spv-wallet-stack/charts/spv-wallet/values.yaml

```diff
@@ -148,6 +156,7 @@ config:
     default_note: SPV Wallet Address Resolution
     # enable paymail domain validation, paymail domain must be in domains list to be valid and that the transaction can be processed
     domain_validation_enabled: true
+    default_from_paymail: ''
```

Comment:
hopefully we will remove it soon, so let's keep it as it was (calculated without option to set it in values).